### PR TITLE
Added Item cntr

### PR DESCRIPTION
**Pull request by AbelG101**
## Changes
- Added meal counter function.
- Removed hard-coded value on the amount of meal.
- Meal amount is loaded dynamically from the mealCounter function.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,7 +1,24 @@
...
+window.onload = async () => {
+  await loadDataFromAPI();
+  countMeals();
+};
+
+document.addEventListener('click', (e) => {
+  if (!e.target.matches('.like-btn')) {
+    return;
+  }
+  if (e.target.matches('.like-btn')) {
```
Comment (binhussen) [inline src/index.js:16]: kindly One conditional statement is enough here.

File: src/modules/countMeals.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const countMeals = () => {
+  const amount = document.querySelector('.item-amount');
+  const items = document.querySelectorAll('.card');
+  let cntr = 0;
+  items.forEach(() => {
+    cntr += 1;
+  });
+  amount.textContent = `(${cntr})`;
+};
```
Comment (binhussen) [inline src/modules/countMeals.js:9]: please you can get number of meals from meals.length instead of counting cards

**Review by binhussen — APPROVED** [review]
### Hi @AbelG101 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions
- please you can get number of meals from meals array of object by meals.length  instead of counting cards


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: src/modules/countMeals.js (hunk 2 above)
Comment (AbelG101) [reply src/modules/countMeals.js:9]: Hi @binhussen that's part of the requirement of the project, to count the items.

**Pull request merged**